daemon/src/monitor.rs

```diff
@@ -490,6 +506,31 @@ impl MonitorParams {
     }
 }
 
+fn map_cets(
+    cets: HashMap<String, Vec<model::cfd::Cet>>,
+    script_pubkey: Script,
+) -> HashMap<String, Vec<Cet>> {
+    cets.iter()
+        .map(|(event_id, cets)| {
+            (
+                event_id.clone(),
+                cets.iter()
+                    .map(
+                        |model::cfd::Cet {
+                             tx, range, n_bits, ..
+                         }| Cet {
+                            txid: tx.txid(),
+                            script: script_pubkey.clone(),
+                            range: range.clone(),
+                            n_bits: *n_bits,
+                        },
```

Comment:
Could implement `from` or a different constructor to remove the low-level bits from this.